packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -649,7 +651,7 @@ export class PaginatedMessage {
 	 * });
 	 * ```
 	 */
-	public addAsyncPageEmbed(embed: EmbedBuilder | ((builder: EmbedBuilder) => Promise<EmbedBuilder>)): this {
+	public addAsyncPageEmbed(embed: EmbedResolvable | ((builder: EmbedBuilder) => Awaitable<EmbedResolvable>)): this {
```

Comment:
Revert this to `Promise`, this is explicitly `addAsyncPageEmbed`, not `addPageEmbed`.

Reply:
The reason why I made it `Awaitable` is because the method supports non-promises, e.g. a callback that doesn't always return a `Promise`, even if niche.